ci/infra/testrunner/tests/test_node_reboot.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import time
+
+logger = logging.getLogger("testrunner")
+
+
+def test_reboot_master(deployment, platform, skuba, kubectl):
+    node_up = False
+    # Make sure there is at least two masters
+    assert skuba.num_of_nodes('master') > 1
+
+    logger.info('Wait for all the nodes to be ready')
+    kubectl.run_kubectl('wait --for=condition=Ready node --all --timeout=5m')
+
+    logger.info('Rebooting master 1')
+
+    try:
+        platform.ssh_run('master', 1, 'sudo reboot')
+    except RuntimeError as ex:
+        logger.debug(ex)
+
+    # Give the reboot a bit to start
+    time.sleep(5)
+
+    start_time = time.time()
+    run_time = 0
+    while run_time < 300:
+        try:
+            platform.ssh_run('master', 1, 'echo hello')
+            kubectl.run_kubectl('wait --for=condition=Ready node --all')
```

Comment:
When you reboot a node it won't go into `NotReady` state that fast. This test will always pass (if ssh starts).
I had this in mind for testing reboot, but I don't have code ready yet (did not test it):
```
# we should somehow check that nodes are really rebooting
1. reboot node && sleep 30

# first problem is that nodes won't go to NotReady during reboot (it's too fast)
# second problem is that pods show error and start restarting only !AFTER! ssh is back
2. wait until ssh comes back && sleep 30

# verify that reboot did not break things
3. check all pods are running
```

Reply:
As for checking the sss is back you can use the approach I suggested before, similar to the one used in tf during deployment